set.mm

```diff
@@ -423544,6 +423582,7 @@ number of vertices in a friendship graph is (k(k-1)+1), see
 $( End $[ set-graphth.mm $] $)
 
```

Comment:
```
$( End $[ set-tarskigeom.mm $] $)
$( Beginn $[ set-graphth.mm $] $)
```

are missing!?

Reply:
The start and end tarskigeom and graphth tags already exist

Reply:
It's because these files were already separate before this pr, so I didn't actually change anything